tests/test_features.py

```diff
@@ -23,7 +14,5 @@ def test_show_features(duthost):
     pytest_assert(succeeded, "failed to obtain feature status")
     for cmd_key, cmd_value in features_dict.items():
         feature = str(cmd_key)
-        status_out = duthost.shell('/usr/bin/redis-cli -n 4 hgetall "FEATURE|{}"'.format(feature), module_ignore_errors=False)['stdout_lines']
-        redis_value = get_status_redisout(status_out)
-        status_value_expected = str(cmd_value)
-        assert str(redis_value) == status_value_expected, "'{}' is '{}' which does not match with config_db".format(cmd_key, cmd_value)
+        redis_value = duthost.shell('/usr/bin/redis-cli -n 4 hget "FEATURE|{}" "state"'.format(feature), module_ignore_errors=False)['stdout']
+        assert redis_value.lower() == cmd_value.lower(), "'{}' is '{}' which does not match with config_db".format(cmd_key, cmd_value)
```

Comment:
The `redis_value` will have double quote, the comparison will fail.
```
admin@str-msn2700-02:~$ /usr/bin/redis-cli -n 4 hget "FEATURE|teamd" "state"
"enabled"
admin@str-msn2700-02:~$ /usr/bin/redis-cli -n 4 --raw hget "FEATURE|teamd" "state"
enabled
```

Suggest to add `--raw` option for the `redis-cli` command.

Reply:
Thanks. Updated.